pkg/clients/logging/logger.go

```diff
@@ -49,11 +48,11 @@ var LogLevels = []string{
 
 type LoggingClient interface {
 	SetLogLevel(logLevel string) error
-	Debug(msg string, args ...interface{}) error
-	Error(msg string, args ...interface{}) error
-	Info(msg string, args ...interface{}) error
-	Trace(msg string, args ...interface{}) error
-	Warn(msg string, args ...interface{}) error
+	Debug(msg string) error
```

Comment:
The entire point of #739 was adding this functionality.

Reply:
Agreed, see my additional comment on logging/types.go